test/Microsoft.DotNet.Tools.Tests.Utilities/Commands/CacheCommand.cs

```diff
@@ -83,7 +85,7 @@ private string BuildArgs()
                 FrameworkVersionOption);
         }
 
-        private string ProfileProjectOption => string.IsNullOrEmpty(_profileProject) ? "" : $"--entries {_profileProject}";
+        private string ProfileProjectOption => _profileProject.Count == 0  ? "" :  string.Join(" ", _profileProject) ;
```

Comment:
Won't string.Join handle the empty case as well?  I don't think you need the ternary conditional operator here.